articles/azure/azs-how-create-vm-terraform.md

```diff
@@ -142,6 +142,7 @@ The examples that follow show how to create VMs using Terraform. The code change
 | client_secret    | The application password that you have configured your Service Principal Name (SPN) to use.    | <form oninput="result.value=client_secret.value" id="client_secret" style="display:inline;"><input type="text" id="client_secret" name="client_secret" style="display:inline;" placeholder="{applicationPassword}"/></form> |
 | rg_name    | The name of the resource group you wish to create    | <form oninput="result.value=rg_name.value" id="rg_name" style="display:inline;"><input type="text" id="rg_name" name="rg_name" style="display:inline;" placeholder="MyResourceGroup"/></form> |
 | tenant_id    | The tenant ID of your Azure Active Directory tenant domain. It can either be the actual GUID or your Azure Active Directory tenant domain name. | <form oninput="result.value=tenant_id.value" id="tenant_id" style="display:inline;"><input type="text" id="tenant_id" name="tenant_id" style="display:inline;" placeholder="xxxxx-xxxx-xxxx-xxxx-xxxxxxxxxx"/></form> |
+| location    | Name of the Azure/Azure Stack region. | <form oninput="result.value=location.value" id="location" style="display:inline;"><input type="text" id="location" name="location" style="display:inline;" placeholder="frn00006"/></form> |
```

Comment:
"The name of ..." all the others start with "The". Best to keep consistency?

Reply:
I agree - and I made the change